.gitignore

```diff
@@ -14,3 +14,6 @@ vendor-bin/**/composer.lock
 
 # SonarCloud scanner
 .scannerwork
+
+# Generated from .drone.star
+.drone.yml
```

Comment:
This repo still has `.drone.starlark` and `.drone.yml` because if we make `.drone.star` then the auto-generated pipelines on the drone server are to big. I exceeds a (silly) 1MB limit.

So for this repo we do not want this added to `.gitignore`